tests/apps/courses/test_templates_course_detail_rendering.py

```diff
@@ -110,6 +110,12 @@ def test_templates_course_detail_cms_published_content(self):
         )
 
         response = self.client.get(url)
+        content = htmlmin.minify(
+            response.content.decode("UTF-8"),
+            reduce_empty_attributes=False,
+            remove_optional_attribute_quotes=False,
+        )
+
```

Comment:
Kinda like my comment on the removal of the category glimpse title, I'm not sure why this and a few other test changes are added in this PR. It seems unrelated to the other things. I reverted those changes too to only add the icons/title attribute changes. I'll let you handle those in another PR if it was really intended, I'm not sure what this is about :)